src/data/showcases.js

```diff
@@ -1094,6 +1094,15 @@ export const Showcases = [
     source: null,
     tags: [ "daotool"],
   },
+  {
+    title: "Great Empires",
+    description:
+      "Building ancient era metaverse on Cardano Blockchain. Phase 01 : Digital Collectible Card Game, Phase 02 : Metaverse, Utility token of the ecosystem GREMP",
+    preview: require("./showcase/great_empires.png"),
+    website: "https://greatempires.io/",
+    source: null,
+    tags: [ "game", "nft", "nativetoken"],
```

Comment:
```suggestion
    tags: [ "game", "nftproject", "token"],
```
One of the tags you used is from the Builder Tools and the other is the wrong tag value... but these will work 🤓